### PR TITLE
arm osx migration

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
Hi @xorg-libxtst this is ready to be merged. Thanks!

**Comment by Tobias-Fischer** [comment]
Fix #7

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,19 +26,22 @@ requirements:
...
     - posix  # [win]
+    - gnuconfig  # [unix]
     - {{ compiler('c') }}        # [unix]
     - {{ compiler('m2w64_c') }}  # [win]
     - make   # [unix]
   host:
     - xorg-inputproto
-    - xorg-libx11 1.6.*
+    - xorg-libx11 1.6.*  # [not (osx and arm64)]
+    - xorg-libx11 1.7.*  # [osx and arm64]
```
Comment (pkgw) [inline recipe/meta.yaml]: We have been using slightly different boilerplate to do the libx11 pinning, namely:

https://github.com/conda-forge/xorg-libxfixes-feedstock/blob/0b7e78d73fbab3f3d93716b54a618e8b3fb4d055/recipe/meta.yaml#L41-L43

in the `host` section and and

https://github.com/conda-forge/xorg-libxfixes-feedstock/blob/0b7e78d73fbab3f3d93716b54a618e8b3fb4d055/recipe/meta.yaml#L49

in the `run` section. This format should be more future-proof for changes to the pinning. Would you mind changing this section to use the same template, including the same explanatory comment?

**Review by pkgw — CHANGES_REQUESTED** [review]
Thanks for fixing up the bot PR! We've been using a slightly different pattern for the libx11 pinning, which I think is a bit more future-proof — see below.

**Comment by Tobias-Fischer** [comment]
Thanks for the review @pkgw - it should be ready to go now.

**Review by pkgw — APPROVED** [review]
Thanks!

**Pull request merged**